src/shogun/kernel/Kernel.h

```diff
@@ -62,6 +62,7 @@ enum EKernelType
 	K_GAUSSIAN = 30,
 	K_GAUSSIANSHIFT = 32,
 	K_GAUSSIANMATCH = 33,
+	K_GAUSSIANFITC = 34,
```

Comment:
@karlnapf 
Let me mention another reason why not use `CGaussianKernel` as the base class.
The reason is for the `scalar` version of Gaussian ARD kernel, we can use the `dot` operation in `CDotFeatures`, which is implemented in the exiting `CGaussianKernel`.
The `dot` operation can support `sparse` dot operation.

However, for the `vector` and `matrix` versions, I have to first get `Dense` vectors and then use them to compute kernel values.